packages/react-accordion/src/components/AccordionPanel/useAccordionPanel.ts

```diff
@@ -16,15 +16,17 @@ export const accordionPanelShorthandProps: Array<keyof AccordionPanelSlots> = []
  */
 export const useAccordionPanel = (props: AccordionPanelProps, ref: React.Ref<HTMLElement>): AccordionPanelState => {
   const { open } = useAccordionItemContext();
-  const id = useId('accordion-panel-', props.id);
-  const innerRef = React.useRef<HTMLElement>(null);
-  const state: AccordionPanelState = {
+  return {
     open,
-    role: 'region',
-    // 'aria-labelledby': header?.id,
+    // TODO: this is a hack for conformance problems
```

Comment:
Can you please clarify what causes the issue?

Reply:
`AccordionPanel` only renders when the context provided by `AccordionItem` states that it's open, otherwise nothing is rendered.

I couldn't manage to find a way to force open `AccordionPanel`  in tests, therefore I'm spreading `props` to allow the override by props to the control of the open state of the `AccordionPanel`